### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by grandoc**

### CODE REVIEW

File: dev/skeletons/skeleton_class.class.php [hunk 1]
```diff
@@ -262,7 +262,7 @@ function delete($user, $notrigger=0)
 			if (! $notrigger)
 			{
 				// Uncomment this and change MYOBJECT to your own tag if you
-		        // want this action call a trigger.
+		        // want this action calls a trigger.
```
Comment (marcosgdf) [inline dev/skeletons/skeleton_class.class.php:265]: I do think it was correct the old way

Comment (grandoc) [inline dev/skeletons/skeleton_class.class.php:265]: Who(which) is calling > this action ( third person) > take an 's'
I guess :-\

@+
Philippe
Le 20/08/2012 17:55, Marcos García a écrit :

> > ```
> >             // Uncomment this and change MYOBJECT to your own tag if you
> > ```
> > -               // want this action call a trigger.
> > -               // want this action calls a trigger.

## 

Atoo.Net http://www.atoo-net.com  Philippe GRAND
265 rue de la vallée
45160 Olivet
02 38 63 90 20
philippe.grand@atoo-net.com

Comment (marcosgdf) [inline dev/skeletons/skeleton_class.class.php:265]: It doesn't sound good to me... but I don't really know how to check if that's correct or not.
`Uncomment this and change MYOBJECT to your own tag if you want this action to call a trigger`

Comment (grandoc) [inline dev/skeletons/skeleton_class.class.php:265]: http://www.englishtenses.com/third_person_singular :
- _he eats_
- _she eats_
- _it eats_

;-)

@+
Philippe

Le 20/08/2012 19:32, Marcos García a écrit :

> In dev/skeletons/skeleton_class.class.php:
> 
> > @@ -262,7 +262,7 @@ function delete($user, $notrigger=0)
> >             if (! $notrigger)
> >             {
> >                 // Uncomment this and change MYOBJECT to your own tag if you
> > -               // want this action call a trigger.
> > -               // want this action calls a trigger.
> 
> It doesn't sound good to me... but I don't really know how to check if 
> that's correct or not.
> |Uncomment this and change MYOBJECT to your own tag if you want this 
> action to call a trigger|
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/Dolibarr/dolibarr/pull/334/files#r1416197.

## 

Atoo.Net http://www.atoo-net.com  Philippe GRAND
265 rue de la vallée
45160 Olivet
02 38 63 90 20
philippe.grand@atoo-net.com

Comment (marcosgdf) [inline dev/skeletons/skeleton_class.class.php:265]: I'm afraid I still don't agree with you. That would make sense with `This action calls a trigger` but not with `If you want this action to call a trigger`. There's a missing `to` before `call` but I think that's not the main verb.

I know an English teacher and I might have her position about this topic tomorrow. If you can wait till then...

Comment (grandoc) [inline dev/skeletons/skeleton_class.class.php:265]: Le 20/08/2012 19:50, Marcos García a écrit :

> |If you want this action to call a trigger|.
> so we should at least write this and not "if you want this action call a 
> trigger" wich doesn't mean the same

@+

## Philippe

Atoo.Net http://www.atoo-net.com  Philippe GRAND
265 rue de la vallée
45160 Olivet
02 38 63 90 20
philippe.grand@atoo-net.com

Comment (hregis) [inline dev/skeletons/skeleton_class.class.php:265]: and this regis call a philippe ?

Comment (marcosgdf) [inline dev/skeletons/skeleton_class.class.php:265]: Well, I've talked with the English teacher I know and she told me that both sentences are correct, but the most correct is yours, so... sorry :P

**Pull request merged**